cli/normalise.go

```diff
@@ -0,0 +1,90 @@
+package cli
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"go.keploy.io/server/v2/config"
+	toolsSvc "go.keploy.io/server/v2/pkg/service/tools"
+	"go.keploy.io/server/v2/utils"
+	"go.uber.org/zap"
+)
+
+// func NewCmdNormalise(logger *zap.Logger) *Normalise {
+// 	normaliser := normalise.NewNormaliser(logger)
+// 	return &Normalise{
+// 		normaliser: normaliser,
+// 		logger:     logger,
+// 	}
+// }
+
+//	type Normalise struct {
+//		normaliser normalise.Normaliser
+//		logger     *zap.Logger
+//	}
+func init() {
+	Register("normalise", Normalise)
+}
+
+// Normalise retrieves the command to normalise Keploy
+func Normalise(ctx context.Context, logger *zap.Logger, conf *config.Config, serviceFactory ServiceFactory, cmdConfigurator CmdConfigurator) *cobra.Command {
+	var normaliseCmd = &cobra.Command{
+		Use:     "normalise",
+		Short:   "Normalise Keploy",
+		Example: "keploy normalise --path /path/to/localdir --test-set testset --test-cases testcases",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			path, err := cmd.Flags().GetString("path")
```

Comment:
Getting the value from flag is no more needed after refactored changes because viper is dealing with it - https://github.com/keploy/keploy/blob/2554c2f393fa3849b10372bb3449250aee8c901c/cli/provider/cmd.go#L225